modules/viewing-feature-store-objects-in-the-web-based-ui.adoc

```diff
@@ -0,0 +1,93 @@
+:_module-type: PROCEDURE
+
+[id="viewing-feature-store-objects-in-the-web-based-ui_{context}"]
+= Viewing feature store objects in the web-based UI
+
+[role='_abstract']
+You can use the Feature Store Web UI to view all registered features, data sources, entities, and feature services.
+
+.Prerequisites
+
+* You have cluster administrator privileges for your {openshift-platform} cluster.
+* You have installed the OpenShift command-line interface (CLI). See link:https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp-latest-version}/html/cli_tools/openshift-cli-oc#installing-openshift-cli[Installing the OpenShift CLI^].
+* You have installed the Kubernetes Command line tool (kubectl). See https://kubernetes.io/docs/reference/kubectl[Command line tool (kubectl)^].
+
+* You have enabled the Feature Store component, as described in Enabling the Feature Store component.
+
+.Procedure
+
+. In the OpenShift console, click the *Quick Create* (image:images/quick-create-icon.png[]) icon and then click the *Import YAML* option.
+. Verify that your data science project is the selected project.
+. Copy the following code and paste it into the YAML editor:  
++
+[.lines_space]
+[.console-input]
+[source, yaml]
+----
+apiVersion: feast.dev/v1alpha1
+kind: FeatureStore
+metadata:
+  name: sample
+spec:
+  feastProject: my_project
+  services:
+	  ui: {}  # <1>
+----
+<1> Specifies to create a Web UI for the feature store instance.
+
+. Click *Create*.
++
+The Feature Store operator starts a container for the web-based Feature Store UI and creates an OpenShift route that provides the URL so that you can access it. 
```

Comment:
```suggestion
The Feature Store Operator starts a container for the web-based Feature Store UI and creates an OpenShift route that provides the URL so that you can access it. 
```